draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -41,24 +41,28 @@ informative:
 
 --- abstract
 
-QUIC defines a RESET_STREAM frame to abort sending on a stream. When a sender
-resets a stream, it also stops retransmitting STREAM frames for this stream in
-the of packet loss. On the receiving side, there is no guarantee that any data
-sent on that stream is delivered.
+QUIC transport protocol streams provide an ordered and reliable byte stream.
```

Comment:
```suggestion
QUIC transport protocol provides ordered and reliable byte streams.
```
Maybe? Just a matter of taste.

Reply:
Or maybe: QUIC transport protocol provides byte streams that are ordered and reliable.

Reply:
I like the second suggestion, so tweaked it and pushed up